packages/shared/src/components/tooltips/ChangelogTooltip.tsx

```diff
@@ -99,138 +90,122 @@ function ChangelogTooltip<TRef extends HTMLElement>({
   };
 
+  const onToggleUpvote = async () => {
+    await toggleUpvote({ post, origin: Origin.ChangelogPopup });
+  };
+
   return (
-    <BaseTooltip
-      {...props}
-      content={
-        !!post && (
-          <div
-            className="flex relative flex-col whitespace-normal break-words rounded-16 border shadow-2 focus:outline-none w-[24rem] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
-            data-testid="changelog"
+    !!post && (
+      <InteractivePopup
+        {...props}
+        position={InteractivePopupPosition.Unset}
+        className="bottom-0 left-60 mb-6 ml-6 border shadow-2 focus:outline-none w-[24rem] max-w-[360px] max-h-[466px] changelog bg-theme-bg-tertiary border-theme-color-cabbage"
+        data-testid="changelog"
+      >
+        <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
+          <h3
+            className="font-bold typo-title3 text-theme-label-primary"
+            data-testid="changelogNewReleaseTag"
+          >
+            New release
+          </h3>
+          <ModalClose
+            className="right-4"
+            onClick={onModalCloseClick}
+            data-testid="changelogModalClose"
+            buttonSize={ButtonSize.XSmall}
+          />
+        </header>
+        <section className="flex flex-col flex-1 p-5 h-full shrink max-h-full">
+          <Image
+            className="object-cover rounded-lg w-[207px] h-[108px]"
+            alt="Post cover image"
+            src={post.image}
+            fallbackSrc={cloudinary.post.imageCoverPlaceholder}
+            loading="lazy"
+            data-testid="changelogImage"
+          />
+          <h3
+            className="mt-3 font-normal text-theme-label-primary typo-title3"
+            data-testid="changelogTitle"
           >
-            <header className="flex flex-1 items-center py-3 px-4 border-b border-theme-divider-tertiary">
-              <h3
-                className="font-bold typo-title3 text-theme-label-primary"
-                data-testid="changelogNewReleaseTag"
-              >
-                New release
-              </h3>
-              <ModalClose
-                className="right-4"
-                onClick={onModalCloseClick}
-                data-testid="changelogModalClose"
-                buttonSize={ButtonSize.XSmall}
+            {post.title}
+          </h3>
+          <time
+            className="text-theme-label-quaternary typo-callout"
+            dateTime={post.createdAt}
+            data-testid="changelogDate"
+          >
+            {postDateFormat(post.createdAt)}
+          </time>
+          {!!post.summary && (
+            <div
+              className="mt-2 w text-theme-label-tertiary typo-callout"
+              data-testid="changelogSummary"
+            >
+              {post.summary}
+            </div>
+          )}
+          <div className="flex mt-4 font-bold w text-theme-color-salt typo-footnote">
```

Comment:
The following should not be needed here.
font-bold w text-theme-color-salt typo-footnote